.words

```diff
@@ -0,0 +1,9 @@
+8
+behaviour
+coloured
+emph
+gendoc
+hosters
+initialisms
+pluralisation
+repo
```

Comment:
nit: I think "repo" should be part of the default dictionary.

Reply:
I think you are right.

Reply:
I'll change that in another PR and fix this then.